servers/rendering/rasterizer_rd/rasterizer_effects_rd.h

```diff
@@ -640,13 +640,15 @@ class RasterizerEffectsRD {
 		float auto_exposure_grey = 0.5;
 		RID exposure_texture;
 
+		bool use_adjustments = false;
```

Comment:
This is assigned but never used it seems?

Reply:
I thought this was used. Let me check.

Reply:
This is used in:

```c++
// rasterizer_scene_rd.cpp
		if (can_use_effects && env) {
			tonemap.use_adjustments = env->adjustments_enabled;
```

Reply:
Well that's not use, that's assignment, which is what I commented :)

If it's assigned but never actually used as a condition, it seems to be unused.

Reply:
Let me see what happens when I delete it.

Reply:
Deleted.